src/app/interfaces/apiInterfaces.ts

```diff
@@ -30,6 +31,21 @@ export type AuthToken = string;
  * BAW API Item Description
  */
 export type Description = string;
+/**
+ * BAW Notes
+ */
+export type Notes = Blob;
+/**
+ * Attempt to parse notes
+ * @param blob Notes
+ */
+export function notes(blob: string): Blob {
+  if (isUninitialized(blob)) {
+    return undefined;
+  }
+
+  return new Blob([blob], { type: "text/plain" });
```

Comment:
if `notes` is ever not part of a normal object graph (i.e. if `notes` is a string), consider it a bug and mark any "deblobbing" or "reblobbing" as bugs/temporary hacks.

Remove this function - we don't want to normalize the bug